### PR TITLE
:sparkles: Add repository fields to custom-targets-form

### LINKED ISSUE #661: Custom Targets repository upload
- Auto scroll to custom target when show me link is clicked in notification window
- Custom rules field radio button implementation for repositories fields 
- Implement repository fields in custom target form 
- Disable custom rules manual selection type on edit custom target 
- Custom target text overflow in card UX 
- Add custom tag in wizard view of targets 
- Custom rules wizard changes for repository fields

### PR DESCRIPTION
**Pull request by ibolton336**
Closes #661

### CODE REVIEW

**Comment by ibolton336** [comment]
Needs https://github.com/konveyor/tackle2-addon-windup/issues/86 for full functionality

File: client/src/app/pages/applications/analysis-wizard/custom-rules.tsx [hunk 1]
```diff
@@ -32,22 +37,39 @@ import {
...
   const { t } = useTranslation();
 
-  const { watch, setValue } = useFormContext<AnalysisWizardFormValues>();
+  const { watch, setValue, control, getValues, getFieldState, formState } =
+    useFormContext<AnalysisWizardFormValues>();
+
+  const values = getValues();
+
+  console.log("values", values);
+  console.log("errors", formState.errors);
```
Comment (gildub) [inline client/src/app/pages/applications/analysis-wizard/custom-rules.tsx:69]: Do we need those ?

File: client/src/app/pages/applications/analysis-wizard/set-targets.tsx [hunk 2]
```diff
@@ -27,10 +27,14 @@ export const SetTargets: React.FC = () => {
...
 
-  const { watch, setValue } = useFormContext<AnalysisWizardFormValues>();
+  const { watch, setValue, getValues, formState } =
+    useFormContext<AnalysisWizardFormValues>();
   const formTargets = watch("formTargets");
   const formRuleBundles = watch("formRuleBundles");
   const formSources = watch("formSources");
+  const values = getValues();
+  console.log("vals", values);
+  console.log("errors", formState.errors);
```
Comment (gildub) [inline client/src/app/pages/applications/analysis-wizard/set-targets.tsx:37]: Do we need those ?

File: client/src/app/pages/migration-targets/custom-target-form.tsx [hunk 3]
```diff
@@ -72,6 +88,49 @@ export const CustomTargetForm: React.FC<CustomTargetFormProps> = ({
       );
     });
 
+  //Repository options
```
Comment (gildub) [inline client/src/app/pages/migration-targets/custom-target-form.tsx:91]: Redundant ?

**Review by gildub — APPROVED** [review]

**Pull request merged**